CrazyCanvas/Source/CrazyCanvas.cpp

```diff
@@ -76,11 +42,13 @@ void CrazyCanvas::Tick(LambdaEngine::Timestamp delta)
 
 void CrazyCanvas::FixedTick(LambdaEngine::Timestamp delta)
 {
-	UNREFERENCED_VARIABLE(delta);
+	float32 dt = (float32)delta.AsSeconds();
```

Comment:
What is this?

Reply:
A mistake. dt isn't being used, so UNREFERENCED_VARIABLE should still be there.